pkg/evaluation/manager.go

```diff
@@ -209,11 +213,132 @@ func (m *manager) EvaluateResource(ctx context.Context, request *pb.ResourceEval
 }
 
 func (m *manager) GetResourceEvaluation(ctx context.Context, request *pb.GetResourceEvaluationRequest) (*pb.ResourceEvaluationResult, error) {
-	panic("implement me")
+	log := m.logger.Named("GetResourceEvaluation").With(zap.String("id", request.Id))
+
+	searchResponse, err := m.esClient.MultiSearch(ctx, &esutil.MultiSearchRequest{
+		Index: m.indexManager.AliasName(constants.EvaluationsDocumentKind, ""),
+		Searches: []*esutil.EsSearch{
+			{
+				Query: &filtering.Query{
+					Term: &filtering.Term{
+						"_id": request.Id,
+					},
+				},
+			},
+			{
+				Query: &filtering.Query{
+					HasParent: &filtering.HasParent{
+						ParentType: resourceEvaluationRelationName,
```

Comment:
Since this is joined to the parent document, should we be specifying `routing` in the [header metadata](https://github.com/rode/grafeas-elasticsearch/blob/19949ccd5276e484835d5148eb416f48d41d1b2e/go/v1beta1/storage/esutil/client.go#L390-L392)?

Reply:
I had thought that the `Search` method had special handling for when `HasParent` is used, but it seems I was wrong about that.  It actually looks like `Search` doesn't have any way of handling routing.  I think we'll need to update the esutil package to support that.

Reply:
Just kidding, for some reason I thought this comment was left on the `List` implementation instead of `Get`, and `Get` uses `MultiSearch`.  Both `Search` and `MultiSearch` don't seem to have routing support.

However, I tested this locally against a 3 node ES cluster, and this implementation worked correctly with 20 different evaluations, leading me to believe that routing isn't actually required.

It's probably good to add anyways though since I don't understand the specifics of when it's absolutely needed.